packages/switch/src/component.tsx

```diff
@@ -24,18 +24,15 @@ export function Switch({
         aria-labelledby={ariaLabelledBy}
         aria-checked={value}
         onClick={onClick}
-        className={classNames([c.label, switchFocus])}
+        className={classNames([ccSwitch.label, switchFocus])}
         {...attrs}
       >
         <span
-          className={classNames([c.switchTrack, 'top-0', 'left-0'], {
-            [c.switchTrackSelected]: value,
-            [c.switchTrackUnselected]: !value,
-          })}
+          className={classNames([ccSwitch.track, 'top-0', 'left-0', value ? ccSwitch.trackActive : ccSwitch.trackInactive])}
```

Comment:
is there a reason to not add `top-0` and `left-0` to ccSwitch.track classes?

Reply:
Yup, because the React versions of the Switch component is made up of a button and span elements, unlike the Vue equivalent which uses an input and divs. So this is the way it was in Fabric as well...

Reply:
aha okay cool, let's do it like this then!